src/operator/src/tests/partition_manager.rs

```diff
@@ -80,6 +81,14 @@ pub fn new_test_table_info(
         .unwrap()
 }
 
+fn new_test_wal_options_map(
+    regions: Vec<RegionNumber>,
+) -> HashMap<RegionNumber, EncodedWalOptions> {
+    // TODO(niebayes): construct wal options for test.
+    let _ = regions;
+    HashMap::default()
+}
```

Comment:
Maybe we need a method in somewhere to retrieve the options

Reply:
To retrieve the options for a given region?

Reply:
Will add one when necessary.